src/document.c

```diff
@@ -1955,37 +1961,51 @@ static gchar *write_data_to_disk(const gchar *locale_filename,
 		/* Use POSIX API for unsafe saving (GVFS-unsafe) */
 		/* The error handling is taken from glib-2.26.0 gfileutils.c */
 		errno = 0;
-		fp = g_fopen(locale_filename, "wb");
+		fp = g_fopen(locale_filename, "w+b"); // we can not destroy  symbolic/hard links
```

Comment:
@biv91 have you checked that remote filesystems like sshfs support this? 

If you havn't tested it on Linux and OSX and with a large number of different filesystems I would be happier if this was limited to windows only, not imposed on everybody.